Makefile

```diff
@@ -5,4 +5,11 @@ build/image:
 	@echo "Building image u-vocab-api:${VERSION}"
 	@echo " "
 	./gradlew dockerBuildImage --parallel -Pversion=${VERSION}
-	docker tag u-vocab-api:${VERSION} uvocab/u-vocab-api:${VERSION}
\ No newline at end of file
+	docker tag u-vocab-api:${VERSION} uvocab/u-vocab-api:${VERSION}
+
+login/push:
```

Comment:
to standardize the names of "task" on makefile my suggestion for this is `image/push`

Reply:
Ok!! thanks, I'll change